bug/bug_test.go

```diff
@@ -109,3 +109,63 @@ func equivalentBug(t *testing.T, expected, actual *Bug) {
 
 	assert.Equal(t, expected, actual)
 }
+
+func TestListParticipants(t *testing.T) {
+	bug := NewBug()
+
+	rene := identity.NewIdentity("René Descartes", "rene@descartes.fr")
+	leonard := identity.NewIdentity("Leonard Euler", "leonard@euler.ch")
+	bernhard := identity.NewIdentity("Bernhard Reimann", "bernhard@reimann.it")
+
+	participants := []identity.Interface{rene, bernhard}
+
+	createOp := NewCreateOp(rene, time.Now().Unix(), "title", "message", nil)
+	setTitleOp := NewSetTitleOp(leonard, time.Now().Unix(), "title2", "title1")
+	addCommentOp := NewAddCommentOp(bernhard, time.Now().Unix(), "message2", nil)
+	labelChangeOp := NewLabelChangeOperation(leonard, time.Now().Unix(), []Label{"label1"}, []Label{})
+
+	bug.Append(createOp)
+	bug.Append(setTitleOp)
+	bug.Append(addCommentOp)
+	bug.Append(labelChangeOp)
+
+	repo := repository.NewMockRepoForTest()
+
+	err := bug.Commit(repo)
+	assert.Nil(t, err)
```

Comment:
you can use `assert.NoError(t, err)` that will give a better output if something break